tools/gyp/gyp_main.py

```diff
@@ -10,6 +10,8 @@
 # Make sure we're using the version of pylib in this repo, not one installed
 # elsewhere on the system.
 sys.path.insert(0, os.path.join(os.path.dirname(sys.argv[0]), 'pylib'))
+sys.argv.append("-DPYTHON_EXECUTABLE=" + os.path.realpath(sys.executable))
```

Comment:
This change should be upstreamed to https://github.com/refack/GYP

In the context of this PR it's not necessary as there are should no be any direct calls to GYP outside of through `./configure`